src/afl-showmap.c

```diff
@@ -1240,7 +1240,7 @@ int main(int argc, char **argv_orig, char **envp) {
 
     u32 save_be_quiet = be_quiet;
     be_quiet = !debug;
-    fsrv->map_size = 4194304;  // dummy temporary value
+    fsrv->map_size = FS_OPT_MAX_MAPSIZE;  // dummy temporary value
```

Comment:
how about an `if (map_size > 4194304) { fsrv->map_size = map_size; } else { fsrv->map_size = 4194304; }`

Reply:
I think that's ok

Reply:
changed to this variant